docker-compose.yml

```diff
@@ -17,6 +17,8 @@ services:
       - IDAM_S2S_AUTH_TOTP_SECRET=AABBCCDDEEFFGGHH
       - FPL_LOCAL_AUTHORITY_NAME_MAPPING=swansea.gov.uk=>SA;hillingdon.gov.uk=>HN
       - FPL_LOCAL_AUTHORITY_USER_MAPPING=SA=>25,26,27;HN=>28,29
+      - FPL_HMCTS_EMAIL_MAPPING=SA=>FamilyPublicLaw+sa@gmail.com;HN=>FamilyPublicLaw+hn@gmail.com
```

Comment:
I assume later we will have `FPL_CAFCASS_EMAIL_MAPPING`?

We getting more and more properties and I worry that the names are not descriptive enough. 

Maybe we should have something like `FPL_LOCAL_AUTHORITY_CODE_TO_HMCTS_EMAIL_MAPPING`. I also think we should add section to README describing all mappings.